File: src/main/java/org/zenbot/szolnok/timetable/batch/configuration/properties/TimetableProperties.java
package org.zenbot.szolnok.timetable.batch.configuration.properties;

import lombok.Data;
import org.springframework.boot.context.properties.ConfigurationProperties;

@Data
@ConfigurationProperties("timetable")
public class TimetableProperties {
    private TimetableResourceProperties resource;
    private TimetableSelectorProperties selector;
}

File: szolnok-timetable-batch-bus/src/main/kotlin/org/zenbot/szolnok/timetable/batch/bus/configuration/properties/TimetableProperties.kt
package org.zenbot.szolnok.timetable.batch.bus.configuration.properties

import org.springframework.boot.context.properties.ConfigurationProperties

@ConfigurationProperties("timetable")
data class TimetableProperties(
        var resource: TimetableResourceProperties = TimetableResourceProperties(),
        var selector: TimetableSelectorProperties = TimetableSelectorProperties()
)
